### PR TITLE
Enable the test for arithmetic inside a filter

### PR DESCRIPTION
**Pull request by ariya**
This is now working after the feature implementation (#14325)

How to verify:
```
yarn test-cypress-no-build --spec frontend/test/metabase/scenarios/question/notebook.cy.spec.js
```

### CODE REVIEW

**Comment by ariya** [comment]
The test failure is legitimate, but not related to this PR. It's because I've rebased `0.39-improve-custom-expression`. Will work on fixing it.

**Comment by ariya** [comment]
> The test failure is legitimate, but not related to this PR. It's because I've rebased `0.39-improve-custom-expression`. Will work on fixing it.

Problem solved. This PR should be ready for review.

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js [hunk 1]
```diff
@@ -598,6 +598,10 @@ describe("scenarios > question > notebook", () => {
         .click();
 
       cy.contains(/^redundant input/i).should("not.exist");
+
+      cy.findAllByRole("button")
+        .contains("Done")
+        .should("not.be.disabled");
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/question/notebook.cy.spec.js:604]: The "correct" syntax for this custom command is:
`cy.findAllByRole("button", { name: "Done" })`

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js [hunk 2]
```diff
@@ -563,7 +563,7 @@ describe("scenarios > question > notebook", () => {
   });
 
   // TODO: add positive assertions to all 4 tests when we figure out implementation details
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/question/notebook.cy.spec.js:565]: Do we want/need to add this?

I.e. Is there any point in actually clicking the "Done" button, and then clicking on "Visualize" to make sure everything works?

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js:604]: Another reason against using `contains`. This doesn't check the button at all. It checks the div.
image

**Review by nemanjaglumac — CHANGES_REQUESTED** [review]
Left a comment re: `find(All)ByRole` syntax. It should be addressed before merge, imo because we're not checking that the actual button is not disabled at all.

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js:604]: Also, why is there another button check at this line when we had it already just a few lines above?
https://github.com/metabase/metabase/pull/14997/files#diff-7db15743c062b782ece95ce8a874e0510f2c73012d08e1656bd3a0ce1e55c639L595-L598

Comment (ariya) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js]: Good points! I've since fixed the issues.

**Review by nemanjaglumac — APPROVED** [review]
Awesome.
Nice abstractions in these tests. 🎉

**Pull request merged**